security/pkg/nodeagent/cache/secretcache_test.go

```diff
@@ -996,19 +996,12 @@ func TestWorkloadAgentGenerateSecretFromFile(t *testing.T) {
 // TestWorkloadAgentGenerateSecretFromFileOverSds tests generating secrets from existing files on a
 // secretcache instance, specified over SDS.
 func TestWorkloadAgentGenerateSecretFromFileOverSds(t *testing.T) {
-	fakeCACli, err := mock.NewMockCAClient(0, time.Hour)
-	if err != nil {
-		t.Fatalf("Error creating Mock CA client: %v", err)
-	}
 	opt := Options{
-		RotationInterval: 200 * time.Millisecond,
+		RotationInterval: 1 * time.Millisecond,
```

Comment:
Can you add a comment to explain why the RotationInterval is changed from 200 ms to 1 ms?

Reply:
I think we do not need rotation here because we do not set `UseCaClient` to false. I can revert this change

Reply:
If the reason of changing RotationInterval from 200 ms to 1 ms is unclear, it is ok to revert this change.